.gitattributes

```diff
@@ -200,4 +200,9 @@ makefile       text
 ## EXECUTABLES
 *.pyc binary
 
-
+## LINGUST OVERRIDES
+/Dockerfile.dev linguist-language=dockerfile
+/docker-compose linguist-language=dockerfile
+/Dockerfile linguist-language=dockerfile
```

Comment:
Also add other files which are not styled by default, like there are some configuration files not styled add them also